commands/bench-all/lib/bench-all-pallet.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+shopt -s inherit_errexit
+shopt -s globstar
+
+. "$(dirname "${BASH_SOURCE[0]}")/../../utils.sh"
+. "$(dirname "${BASH_SOURCE[0]}")/../../cmd_runner.sh"
+
+get_arg required --pallet "$@"
+PALLET="${out:-""}"
+
+REPO_NAME="$(basename "$PWD")"
+BASE_COMMAND="$(dirname "${BASH_SOURCE[0]}")/../../bench/bench.sh --noexit=true --subcommand=pallet"
+WEIGHT_FILE_PATHS=()
+
+# find all weights files
+for f in **/weights/${PALLET}.rs; do
+  WEIGHT_FILE_PATHS+=("$f")
+done
```

Comment:
Should be equivalent
```suggestion
WEIGHT_FILE_PATHS=()

WEIGHT_FILE_PATHS+=(**/weights/${PALLET}.rs)
```
However, both approaches have an issue if glob has no matches: 
<img width="1728" alt="Screen Shot 2024-01-25 at 13 58 39" src="https://github.com/paritytech/command-bot-scripts/assets/588262/5e21ce79-e430-4446-a18e-01bee097c184">

If that's an edge case you'd like to cover, consider using `find` + `while read`

Reply:
great catch, thanks! it didn't play well with not found results, changed to find/read 👍